exp/api/v1alpha3/machinepool_types.go

```diff
@@ -205,6 +205,7 @@ func (m *MachinePoolStatus) GetTypedPhase() MachinePoolPhase {
 // +kubebuilder:storageversion
 // +kubebuilder:printcolumn:name="Replicas",type="string",JSONPath=".status.replicas",description="MachinePool replicas count"
 // +kubebuilder:printcolumn:name="Phase",type="string",JSONPath=".status.phase",description="MachinePool status such as Terminating/Pending/Provisioning/Running/Failed etc"
+// +kubebuilder:printcolumn:name="Version",type="string",JSONPath=".status.version",description="Kubernetes version associated with this MachinePool"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:name="Version",type="string",JSONPath=".spec.template.spec.version",description="Kubernetes version associated with this MachinePool"

cc @CecileRobertMichon to confirm that's what we want
```

Reply:
correct, we want `.spec.template.spec.version`

Reply:
Done, thank you for the reviews @CecileRobertMichon @vincepri